changelogs/unreleased/gh-7046-fix-nullable-complex-fkeys.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/core
+
+* Fixed a bug in the implementation of complex foreign keys that didn't allow
```

Comment:
Please make it shorter. We usually make one sentence for minor issues.

Reply:
Done